frontend/src/utils/customHooks/useParamsWithoutSlug.js

```diff
@@ -0,0 +1,13 @@
+import { useParams } from 'react-router-dom'
+
+const useParamsWithoutSlug = () => {
+  const { id } = useParams()
+
+  const dashIndex = id.indexOf('-')
+
+  return dashIndex !== -1
```

Comment:
I'm not sure if it's common in React of even JS, but IMO a ternary operator that doesn't easily fit in one line should be changed to if statement.